pabi_procurement/wizard/print_purchase_work_acceptance.py

```diff
@@ -11,12 +12,153 @@ class PrintPurchaseWorkAcceptance(models.TransientModel):
 
     name = fields.Char(
         string="Acceptance No.",
+        required=True,
     )
     acceptance_line = fields.One2many(
```

Comment:
Should be acceptance_line_ids
(Always use "ids" to indicate One2many or Many2many